src/main/java/uk/gov/register/core/RegisterContext.java

```diff
@@ -215,6 +223,14 @@ private PostgresDataAccessLayer getTransactionalDataAccessLayer(Handle handle) {
                 getIndexFunctions());
     }
 
+    private void validate() {
+        try {
+            environmentValidator.validateExistingMetadataAgainstEnvironment(this);
+        } catch (FieldValidationException ex) {
```

Comment:
I believe `EnvironmentValidator.validateExistingMetadataAgainstEnvironment(this)` also throws a `RegisterValidationException` so we should also catch this here. This means the register is inconsistent according to the Register register, whereas the `FieldValidationException` means that the register is inconsistent according to the Field register.

I think the reason that this was missed is that we might want to change the declaration of `EnvironmentValidator.validateExistingMetadataAgainstEnvironment` from:

```
    public void validateExistingMetadataAgainstEnvironment(RegisterContext registerContext) throws FieldValidationException {} 

```

to:

```
    public void validateExistingMetadataAgainstEnvironment(RegisterContext registerContext) throws FieldValidationException, RegisterValidationException {}

```